kie-drools-wb/kie-drools-wb-distribution-wars/src/main/assembly/component-kie-drools-wb-tomcat-7-common.xml

```diff
@@ -98,8 +100,6 @@
           <!-- Errai -->
           <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
           <exclude>WEB-INF/lib/errai-cdi-server-*.jar</exclude>
```

Comment:
@psiroky using CDI 1.1 or higher and Errai 4, you DO need errai-cdi-server, and should NOT have any errai-weld-integration on the classpath.

errai-cdi-client is not needed in the .WAR file in Errai 4.